### PR TITLE
{bio}[intel/2018a] AFNI v18.1.09 + deps

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2583.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/90d3c3798b848bd71c2d278772ffedd6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2578.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8a9b665965a03c30d64aa2c4b26a5fb1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2065.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/56a3fcad2b1d95d3ef0a7b316bc0d30b for a full test report.

File: easybuild/easyconfigs/t/tcsh/tcsh-6.20.00-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ mechanism, job control and a C-like syntax."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [
+    'ftp://ftp.astron.com/pub/%(namelower)s',
+    'ftp://ftp.astron.com/pub/%(namelower)s/old',
+]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/t/tcsh/tcsh-6.20.00-GCCcore-6.4.0.eb]: Missing checksum

**Comment by boegel** [comment]
@vanzod I've reached out to the AFNI developers via https://afni.nimh.nih.gov/contact, let's see if that changes anything...

**Comment by boegel** [comment]
No response from AFNI developers, I'm not sure how to continue with this...

**Comment by boegel** [comment]
Oh look what I found! https://github.com/afni/afni/releases

**Comment by boegel** [comment]
@vanzod Please retest?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2013.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d08132289c7f19cc2c9f4ac46bd5ab95 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
cermis - Linux debian 9.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/afd675466da6b282f742837aa62c0a86 for a full test report.

**Comment by vanzod** [comment]
It looks like it needs `M4` as builddep:
```
m4 -B10000 putget.m4 >putget.c
/bin/sh: 1: m4: not found
../rules.make:45: recipe for target 'putget.c' failed
```

File: easybuild/easyconfigs/a/AFNI/AFNI-18.1.09-intel-2018a-Python-3.6.4.eb [hunk 2]
```diff
@@ -22,6 +22,7 @@ checksums = [
     '8b739ddc09d6e398ac7fa86d89f6a02f26f2b58b17caea627d5c07de5282aab2',  # AFNI-18.1.09_omp-pragma-statement-fix.patch
 ]
 
+builddependencies = [('M4', 'M4-1.4.18')]
```
Comment (vanzod) [inline easybuild/easyconfigs/a/AFNI/AFNI-18.1.09-intel-2018a-Python-3.6.4.eb]: There is a typo in the version...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2010.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/95fb229b93bb26ba03b6b5e3a967b768 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
cermis - Linux debian 9.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/dd4e352d495f24e97f08e6579e15fd77 for a full test report.

**Comment by vanzod** [comment]
Now what is missing is `libXp` in `X11`...

**Comment by boegel** [comment]
@vanzod Looks like we'll need to backport #6873 to `X11-20180131-GCCcore-6.4.0.eb` for that...

**Comment by boegel** [comment]
@vanzod `libXp` is added to `X11` bundle used by `2018a` easyconfigs in #7062...

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
cermis - Linux debian 9.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/baaf96033d17248fe69073d601fdbf46 for a full test report.

**Comment by vanzod** [comment]
It looks like the error is now due to the lack of `libtracktools`.
Also, does your build produce `3dkmeans`? It is missing in mine.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.11
See https://gist.github.com/c709caad1b08a280ea89a8f439c402c6 for a full test report.

**Comment by boegel** [comment]
@vanzod Ping on a detailed build log for this?

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
cermis - Linux debian 9.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/cf0f01c24fe6e8d6e4408354ee122a80 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**